### DISCUSSION TITLE
How to configure transport layer protocol (TCP, UDP etc) and application ports in security policy ?

### DISCUSSION THREAD

**Discussion by lkpatel** [strongswan/strongswan#2383] [Q&A] [answered]
Hi, 
Is there any way to configure transport layer protocol (TCP, UDP etc) and application ports in security policy ? I have a requirement where I want to encrypt only TCP and port 80 traffic ?

**Comment by tobiasbrunner** [comment, marked as answer]
Please read [the documentation](https://docs.strongswan.org/docs/5.9/swanctl/swanctlConf.html) for the `local|remote_ts` options.
